oci/layout/oci_transport_test.go

```diff
@@ -124,6 +136,10 @@ func TestNewReference(t *testing.T) {
 
 	_, err = NewReference(tmpDir+"/has:colon", imageValue)
 	assert.Error(t, err)
+
+	// Test private newReference
+	_, err = newReference(tmpDir, imageValue, 1)
+	assert.Error(t, err)
 }
```

Comment:
LGTM. It might be possible to reduce repetition of the two cases, but it’s not at all clear that it would be more readable, and it’s not probably not worth the effort.
```suggestion
}
```